torchelie/optim.py

```diff
@@ -40,8 +40,6 @@ def step(self, closure=None):
                 # Perform stepweight decay
                 p.data.mul_(1 - group['lr'] * group['weight_decay'])
 
-                state = self.state[p]
-
```

Comment:
I don't feel quite good about removing this one actually. Commenting this line would feel better. I don't want to forget that there is an inherited `state` dict that could be extremely helpful.

Reply:
In which way could it be extremely helpful? Could a comment about that as a reminder instead of an unused variable be more fitting? :slightly_smiling_face: